models/migrations/v1_22/v283.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+package v1_22 //nolint
+
+import (
+	"errors"
+	"fmt"
+
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/setting"
+
+	"xorm.io/xorm"
+)
+
+func ExpandHashReferencesToSha256(x *xorm.Engine) error {
```

Comment:
Use `ModifyColumn` method but not write it yourself.

Reply:
Thanks, will do.